common/oe_host_string.h

```diff
@@ -38,11 +43,18 @@ int oe_strncmp(const char* s1, const char* s2, size_t n)
 
 /* host already has an oe_strlcat implementation */
 
+#if defined(_WIN32)
```

Comment:
I think that this should be checking if defined(_MSC_VER) not _WIN32, since I think the warnings are compiler-specific, not platform-specific.